app/components/TimeSeriesChart.tsx

```diff
@@ -125,13 +125,26 @@ export default function TimeSeriesChart({
   maxValue,
   unit,
 }: Props) {
+  // We use the largest data point +20% for the graph scale
+  // clamping at `maxValue` (if set) which is usually overall capacity
+  const calculatedMaxValue = useMemo(() => {
+    if (!maxValue) return null
+    if (!rawData) return maxValue
+    const dataMax = Math.max(...rawData.map((datum) => datum.value))
+    const unroundedValue = Math.min(maxValue, dataMax * 1.2)
```

Comment:
As long as we don't have the capacity number nailed down, I'm wary of picking `maxValue` as the max if `dataMax` is actually bigger. In the unfortunate case where `dataMax` is bigger than `maxValue`, we have to set the chart max to at least `dataMax`, otherwise you're not seeing all the data. Ideally in that situation we would also show a dotted flat line or something at `maxValue` too, though that can be a separate change. We'd have to explain it somehow too.

I came up with this after banging my head against this for bit. Does it make any sense? 

|Situation| Exact range | Desired max |
|---|---|---|
| Data way less than cap | `dataMax * 1.2 < maxValue` | `dataMax * 1.2` |
| Data close to cap but still under | `dataMax < maxValue < dataMax * 1.2 ` | `maxValue` |
| Data over cap (pathological but possible) | `maxValue < dataMax` | `dataMax * 1.2` |

The middle one is the odd one out. I think that's what's messing me up. What if we _always_ used `dataMax * 1.2` as the top edge of the chart, and then if `maxValue` happens to be inside the range of the chart, we show it as a dotted line? Or maybe not always `dataMax * 1.2` — maybe if `dataMax` is anywhere near `maxValue`, like at least half of it, we make it so the `maxValue` line is shown.

Reply:
That sounds about right for me. Setting the top of the graph to the capacity doesn't really clearly communicate that if it's not being applied regardless of usage. So some other indicator is probably best.

Whilst the max capacity is still fuzzy, we could probably just omit it and the logic around keeping the `maxValue` line visible. Happy to just change to `dataMax * 1.2` for now.

Reply:
I think that would work. We'd scrap `maxValue` altogether in that case, right?

Reply:
For now yeah, we can revisit when we have updated capacity values. Looks like we can use Recharts reference lines when thats ready: https://recharts.org/en-US/examples/LineChartWithReferenceLines